packages/server/eslint.config.js

```diff
@@ -0,0 +1,12 @@
+export default [
+	{
+		name: "@svelte-docgen/server",
+		languageOptions: {
+			globals: {
+				bun: "readonly",
+				deno: "readonly",
+				node: "readonly",
+			},
+		},
+	},
+];
```

Comment:
@ciscorn Can I have your help on this one? I'm not up to date with the new ESLint `v9` config setup, especially for monorepo.

Adding this file silences the errors like these:

- https://github.com/svelte-docgen/svelte-docgen/pull/37/files#r1887848947
- https://github.com/svelte-docgen/svelte-docgen/pull/37/files#r1887848952

**only** from the package level  _(
inside the `./packages/server`). It doesn't fail when running:

```sh
pnpm eslint .
```

However, using the same command at the root level it fails, so does CI.

Reply:
I’m not super familiar with ESLint either, but I believe ESLint only looks for the nearest `eslint.config.js`. In the current setup, the easiest solution might be to move your configuration to the root-level `eslint.config.js`. I don’t think it would cause practical issues.

(Alternatively, we could tweak our scripts in `package.json` to run ESLint individually for each package, which might work better, especially if we’re trying to take advantage of Turborepo.)